web/pom.xml

```diff
@@ -0,0 +1,175 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>pro.taskana</groupId>
+  <artifactId>taskana-web</artifactId>
+  <version>0.1.2-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>taskana web</description>
+  <url>http://taskana.pro</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>Holger Hagen</name>
+      <email>holger.hagen@novatec-gmbh.de</email>
+      <organization>NovaTec Consulting GmbH</organization>
+      <organizationUrl>https://www.novatec-gmbh.de</organizationUrl>
+    </developer>
+  </developers>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
```

Comment:
this is such a minimal jar, we do not need a parent - at least spring boot - here..